src/weather_model_graphs/create/base.py

```diff
@@ -198,7 +205,52 @@ def connect_nodes_across_graphs(
     # Determine method and perform checks once
     # Conditionally define _find_neighbour_node_idxs_in_source_mesh for use in
     # loop later
-    if method == "nearest_neighbour":
+    if method == "containing_rectangle":
+        if (
+            max_dist is not None
+            or rel_max_dist is not None
+            or max_num_neighbours is not None
+        ):
+            raise Exception(
+                "to use `containing_rectangle` you should not set `max_dist`, `rel_max_dist`or `max_num_neighbours`"
+            )
+        assert (
+            "dx" in G_source.graph and "dy" in G_source.graph
+        ), "Source graph must have dx and dy properties to connect nodes using method containing_rectangle"
+
+        # Connect to all nodes that could potentially be close enough,
+        # which is at a relative distance of 1
+        rad_graph = connect_nodes_across_graphs(
+            G_source, G_target, method="within_radius", rel_max_dist=1.0
```

Comment:
Why is this 1.0 and not `srt(dx^2 + dy^2)`?

Reply:
Good question. This is relative distance, which is relative to the longest edge in (the bottom level of) the source and target graphs. These longest edges will be the diagonal edges in each rectangle in the mesh graph, which have the length ` sqrt(dx^2 + dy^2)`. So that is exactly the distance of rel_max_dist=1.0. 

I added a bit of a clarification about this, and also realized that there was nothing describing the `rel_max_dist` argument in the docstring, so fixed that as well.